client/configuration.go

```diff
@@ -83,12 +85,19 @@ func (c Configuration) internal(basePath string) (*internal.Configuration, error
 
 	userAgent := fmt.Sprintf("bloxone-%s/%s", sdkIdentifier, version)
 
-	return &internal.Configuration{
+	ic := &internal.Configuration{
 		DefaultHeader:    defaultHeaders,
 		UserAgent:        userAgent,
 		Debug:            false,
 		OperationServers: nil,
 		Servers:          []internal.ServerConfiguration{{URL: cspURL}},
 		HTTPClient:       httpClient,
-	}, nil
+		DefaultTags:      make(map[string]string),
+	}
+	// Add default tags set
+	if c.DefaultTags != nil {
+		ic.AddDefaultTags(c.DefaultTags)
+	}
+
```

Comment:
Why do you need this indirection? Can't you just do directly `ic.DefaultTags = c.DefaultTags` ?

Reply:
Yes, we can. The indirection was added to support `system tag`. I will remove it.